.circleci/config.yml

```diff
@@ -1066,6 +1066,9 @@ jobs:
           name: Build and test Datadog::Php::Components (ASan)
           command: |
             export PATH="/opt/cmake/<< parameters.cmake_version >>/bin:$PATH"
+            if [ -f "/opt/libuv/lib/pkgconfig/libuv.pc" ] ; then
```

Comment:
It is not clear to me when the file is expected not to be present. In other words, it is ok to check or we should enforce existence of the file always when we build?

Reply:
At the moment it only exists on Debian, because I've only updated those images. Eventually it should always exist, or else our build images are bad.

Reply:
thanks 👍🏻